scripts/e2e-widget.js

```diff
@@ -46,17 +48,20 @@ async function runNotebook(name) {
 
 function convertNotebook() {
   console.log("Converting notebook");
-  const { status, stderr } = spawnSync(
-    "jupyter",
-    ["nbconvert", path.join(baseDir, `${filePrefix}*.ipynb`), "--to", "script"],
-    {
-      stdio: "inherit"
+  for (var fileName of fileNames) {
+    console.log(`Converting notebook  ${fileName}\r\n`);

```

Comment:
Do we need to remove these console.log() statements?

Reply:
These console logs will be displayed in cmd before while launching tests. This is helpful in knowing what is happening in the background while tests are launching.